packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -81,6 +82,8 @@ const Tooltip: React.FC<TooltipOwnProps> = (props) => {
         } else if (state === 'closed') {
           setStateAttribute('closed');
         }
+      } else {
+        setStateAttribute('closed');
```

Comment:
The key fix here was this to ensure only one tooltip remain open at once because we had an issue where the state attribute was the same on all instances.

I thought I had fixed it in previous PRs by adding the check for the tooltip id, but we also want to make sure if it's not this particular tooltip, it should be considered closed.